Keas.Mvc/ClientApp/components/Workstations/RevokeWorkstation.tsx

```diff
@@ -68,7 +75,14 @@ export default class RevokeWorkstation extends React.Component<IProps, IState> {
         if (!this.props.selectedWorkstation) {
             return null;
         }
-        await this.props.revokeWorkstation(this.props.selectedWorkstation);
+        this.setState({ submitting: true });
+        try {
+            await this.props.revokeWorkstation(this.props.selectedWorkstation);
+        } catch (err) {
+            this.setState({ submitting: false });
```

Comment:
So once you take out the default error log, are you going to come back here and add error toast for each case like this?

Reply:
or should you add in toast fail/success notifications here?

Reply:
@srkirkland I do the toasts at the actual API call, so in `WorkstationContainer` in this case. then that throws an error that is caught in each modal, so that the modal doesn't close.

Reply:
ah ok, yea that works too